docs: add a guide for querying ClickHouse-backed logs (#47343)
supabase/supabase: apps/docs/content/guides/telemetry/querying-supabase-logs.mdx

```sql
select timestamp, event_message
from logs
where source = 'edge_logs'
order by timestamp desc
limit 100;
```

## The logs table

Each row has a small set of top-level columns. Everything specific to a service lives in `log_attributes`.

| Column           | Type                | Description                                               |
| ---------------- | ------------------- | --------------------------------------------------------- |
| `id`             | string              | Unique log identifier.                                    |
| `timestamp`      | datetime            | When the log was produced, in UTC.                        |
| `event_message`  | string              | The raw log line.                                         |
| `severity_text`  | string              | Log level, when the source sets one.                      |
| `source`         | string              | The service the log came from. Always filter on this.     |
| `log_attributes` | Map(String, String) | Structured fields for the source, keyed by a dotted path. |

> The `timestamp` is a `DateTime64` in UTC, formatted like `2026-06-22T09:34:06.215000` (ISO 8601 with microseconds). You can order and compare it directly. The Logs Explorer already applies the selected time range, so you rarely need a `timestamp` filter of your own.

Author: ```suggestion
The `timestamp` is a `DateTime64` in UTC, formatted like `2026-06-22T09:34:06.215000` (ISO 8601 with microseconds). You can order and compare it directly. The Logs Explorer already applies the selected time range, don't need to write a `timestamp` filter of your own.
```

Author: Applied in 101cf9b1b1.

## Sources

Set `source` to the service you want:

- `edge_logs`: API gateway requests and responses.
- `postgres_logs`: database statements and errors.
- `auth_logs`: authentication and authorization activity.
- `function_edge_logs`: edge function requests and responses.
- `function_logs`: `console` output from inside edge functions.
- `storage_logs`: object upload and retrieval activity.
- `realtime_logs`: Realtime client connections.

The Logs Explorer **Field Reference** drawer lists every source and its fields.

## Reading fields from log_attributes

`log_attributes` is a map from a string key to a string value. Read a field with bracket access:

```sql
select
  log_attributes['request.method'] as method,
  log_attributes['request.path'] as path,
  log_attributes['response.status_code'] as status
from logs
where source = 'edge_logs'
```

There are no unnesting joins. Where BigQuery nested a field under `metadata.request.method`, ClickHouse stores it at `log_attributes['request.method']`.

### Casting numeric fields

Map values are strings. To compare or aggregate a numeric field, wrap it in `toInt32OrZero`:

```sql

> select count(*) as server_errors

Author: we don't support *

Author: Fixed in 101cf9b1b1 — replaced both `count(*)` examples with `count()` since the logs endpoint doesn't support the `*` wildcard.
